### DISCUSSION TITLE
Anybody got private preview access to Github Copilot X?

### DISCUSSION THREAD

**Discussion by nirzaf** [community/community#51475] [Copilot Conversations]
### Select Topic Area

Question

### Body

Anybody got private preview access to Github Copilot X? Just curious to know

**Comment by shomos555** [comment]
I also want access to testing copilot x.((( But it seems that the waiting list is not moving. I am writing a project in flutter for smartphones, and I am very weak in programming. I have already written half of it practically using GPT 4, but it has become unbearably laggy and stupid So I would test your new copilot properly.
